plugin-core/src/main/java/com/salesforce/bazel/eclipse/classpath/BazelClasspathContainer.java

```diff
@@ -98,14 +98,20 @@
     
     private ImplicitDependencyHelper implicitDependencyHelper = new ImplicitDependencyHelper();
     
+    public BazelClasspathContainer(IProject eclipseProject)
+            throws IOException, InterruptedException, BackingStoreException, JavaModelException,
+            BazelCommandLineToolConfigurationException {
+        this(eclipseProject, BazelPluginActivator.getResourceHelper());
+    }
     
-    public BazelClasspathContainer(IProject eclipseProject, IJavaProject eclipseJavaProject)
+    BazelClasspathContainer(IProject eclipseProject, ResourceHelper resourceHelper)
```

Comment:
@plaird I am not actually sure whether this is the right pattern, or whether you intended all helper access to always go through the static methods on BazelPluginActivator.  The change here just adds a ctor intended for tests, so that a mocked ResourceHelper can be passed in.  Or should the mocked ResourceHelper be set on BazelPluginActivator instead?

Reply:
Yes, good question. #16 is the work to rework wireup. Right now, the activator is acting like a Spring context, which isn't right. I am fine with what you have done here.